src/service/performance-impl.js

```diff
@@ -172,18 +172,31 @@ export class Performance {
     if (!this.win.PerformancePaintTiming) {
       return;
     }
+    // Chrome doesn't implement the buffered flag for PerformanceObserver.
+    // That means we need to read existing entries and maintain state
+    // as to whether we have reported a value yet, since in the future it may
+    // be reported twice.
+    // https://bugs.chromium.org/p/chromium/issues/detail?id=725567
+    let recordedFirstPaint = false;
+    let recordedFirstContentfulPaint = false;
+    const processEntry = entry => {
+      if (entry.name == 'first-paint' && !recordedFirstPaint) {
+        this.tickDelta('fp', entry.startTime + entry.duration);
+        recordedFirstPaint = true;
+      }
+      else if (entry.name == 'first-contentful-paint'
+          && !recordedFirstContentfulPaint) {
+        this.tickDelta('fcp', entry.startTime + entry.duration);
+        recordedFirstContentfulPaint = true;
```

Comment:
Don't want to immediately flush anymore?

Reply:
Thanks for the catch. I forgot to hit save on adding it back.

We added this recently, but in practice it didn't help (because it always fired before onload which flushes).